### PR TITLE
SOC2 CI-100 - test environment for MIs described in handbook

### PR DESCRIPTION
**Pull request by filiphaftek**
Description of MI test instances.

### CODE REVIEW

**Review by sanderginn — APPROVED** [review]

File: content/departments/product-engineering/engineering/cloud/devops/managed/index.md [hunk 1]
```diff
@@ -88,12 +88,23 @@ We do not offer Kubernetes managed instances today as this introduces some compl
...
 For each type of Managed Instances (v1.0 and v.1.1), Souregraph maintains separate test environments:
 
 - for v1.0 - [dev instance](https://devmanaged.sourcegraph.com/)
 - for v1.1 - [rctest instance](https://rctest.sourcegraph.com/)
 
-Before performing any changes on customer instances (upgrade, configuration modification etc.), the change is first applied and tested on test instance.
+Internal instances are created for various testing purposes:
+
+- testing changes prior to the montly upgrade on customer instances, e.g <https://devmanaged.sourcegraph.com>
+- testing siginificant opertional changes prior applying to customer instances
```
Comment (sanderginn) [inline content/departments/product-engineering/engineering/cloud/devops/managed/index.md]: few typos
```suggestion
- testing changes prior to the monthly upgrade on customer instances, e.g <https://devmanaged.sourcegraph.com>
- testing significant operational changes prior to applying to customer instances
```

File: content/departments/product-engineering/engineering/cloud/devops/managed/index.md [hunk 2]
```diff
@@ -88,12 +88,23 @@ We do not offer Kubernetes managed instances today as this introduces some compl
...
+- short-lived instances for product teams to test important product changes. Notes: any teammate may request a managed instance through our [request process](./index.md#managed-instance-requests)
+
+### Customer instances
+
+All customer instances are considered production enviornment and all changes applied to these customers should be well-tested in internal environment.
```
Comment (sanderginn) [inline content/departments/product-engineering/engineering/cloud/devops/managed/index.md]:
```suggestion
All customer instances are considered part of the production environment and all changes applied to these customers should be well-tested in the dev/preprod environment.
```

**Review by sanderginn — APPROVED** [review]
Small suggestions

File: content/departments/product-engineering/engineering/cloud/devops/managed/index.md [hunk 3]
```diff
@@ -83,6 +84,29 @@ These managed Docker Compose deployments can scale up to the largest GCP instanc
 
 We do not offer Kubernetes managed instances today as this introduces some complexity for us in terms of ongoing maintenance and overhead, we may revisit this decision in the future.
 
+### Environments
```
Comment (filiphaftek) [inline content/departments/product-engineering/engineering/cloud/devops/managed/index.md:87]: @michaellzc I moved your docs regarding testing environment from operations to main MI page (agreed with Raf that it's very important so should live on first page) and added some small extension for v1.0 & v1.1 - hopefully I moved all correct.

Comment (michaellzc) [reply content/departments/product-engineering/engineering/cloud/devops/managed/index.md:87]: yep! this looks good to me

**Review by michaellzc — APPROVED** [review]

**Pull request merged**